### PR TITLE
Bug wrong count bulk upload dialog when runtime exception

### PR DESCRIPTION
**Pull request by Jonkno**
Fix for the bug by changing the way the syncCountAfterUpload method works. Now the method validates if something went wrong by comparing the successfullTargetCount/failedTargetCount with the total count. The process bar for the upload is also set to 100% if something went wrong.

Reviewer: @melanieretter and @schabdo 

Signed-off-by: Jonathan Philip Knoblauch <JonathanPhilip.Knoblauch@bosch-si.com>

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 1]
```diff
@@ -207,7 +207,7 @@ private void readFileStream(final InputStream tempStream) {
                 eventBus.publish(this, new TargetTableEvent(TargetComponentEvent.BULK_UPLOAD_PROCESS_STARTED));
                 while ((line = reader.readLine()) != null) {
                     innerCounter++;
-                    readEachLine(line, innerCounter, totalFileSize);
+                    readEachLine(line, innerCounter, totalNumberOfLines);
```
Comment (schabdo) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: It makes sense to rename the variable to `totalNumberOfLines`, but than please rename the method parameter `totalFileSize` as well

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 2]
```diff
@@ -193,11 +193,11 @@ public void run() {
 
         private void readFileStream(final InputStream tempStream) {
             String line;
+            final long totalNumberOfLines = getTotalNumberOfLines();
```
Comment (schabdo) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: Do you really have the necessity for type `long`? If so please be consequent and change type of parameter in `readEachLine` as well

Comment (Jonkno) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: Yes, the parameter type in `readEachLine` has allways been `long.` I don't know way it had been converted to double. Didn't make any sense to me.

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java (hunk 1 above)
Comment (Jonkno) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: fixed

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java [hunk 3]
```diff
@@ -269,7 +269,7 @@ private void deleteFile() {
             tempFile = null;
         }
 
-        private void readEachLine(final String line, final double innerCounter, final double totalFileSize) {
+        private void readEachLine(final String line, final double innerCounter, final double totalNumberOfLines) {
```
Comment (michahirsch) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java]: `totalNumberOfLines` does not have to be a `double`, right? maybe can change the type of the variable then it does not have to be auto-boxed to `double`.

**Pull request merged**